File: RepoTransBench/target_projects/Java/C#/mohak1712_UberUX/public_tests/MapActivityPublicTest.cs
using Moq;
using Xunit;

namespace UberUX.Tests.Public
{
    public class MapActivityPublicTest
    {
        private UberUX.MapActivity activity;

        public MapActivityPublicTest()
        {
            var mock = new Mock<UberUX.MapActivity> { CallBase = true };
            activity = mock.Object;
        }

        [Fact]
        public void TestOnCreateExecutesWithoutCrash_Public()
        {
            var bundle = new System.Collections.Generic.Dictionary<string, object>();
            bundle["public_test_double"] = 88.88d;
            activity.onCreate(bundle);
            Assert.NotNull(activity);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/mohak1712_UberUX/public_tests/map_activity_public_test.go
package public_tests

import (
	"testing"
	"mohak1712_uberux/tests"
	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/mock"
)

func TestMapActivityOnCreateExecutesWithoutCrashPublic(t *testing.T) {
	a := new(tests.MapActivity)
	a.On("OnCreate", mock.Anything).Return()
	bundle := tests.Bundle{"public_test_double": 88.88}
	a.OnCreate(bundle)
	assert.NotNil(t, a)
}